main/glossary/README.md

```diff
@@ -459,18 +477,21 @@ For more information, see the [JavaScript Distributed Programming Guide](/guides
 
 ## Proposal
 
-Proposals are records with `give`, `want`, and `exit` properties. [Offers](#offer) must include a proposal, which states
-what asset you want, what asset you will give for it, and how/when the offer maker can cancel the offer
-(see [Exit Rule](#exit-rule) for details on the last). For example:
+Proposals are records with `give`, `want`, and/or `exit` properties respectively
+expressing [offer](#offer) conditions regarding what assets will be given,
+what must be received in exchange to satisfy offer safety, and
```

Comment:
Possible misunderstanding that offer safety means you'll get what you said you want. At least make "offer safety" and explicit link to that glossary entry. Avoiding the possible misunderstanding would be even better.